### PR TITLE
Fix mac build to have `_tkinter`

### LINKED ISSUE #102: Mac build doesn't work
```yaml
Moosems-MBP:~ Moosems$ /Volumes/Minesweeper/minesweeper.app/Contents/MacOS/minesweeper ; exit;
Traceback (most recent call last):
  File "minesweeper.py", line 3, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 540, in exec_module
  File "tkinter/__init__.py", line 37, in <module>
ModuleNotFoundError: No module named '_tkinter'
[10500] Failed to execute script minesweeper
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...27 completed.

[Process completed]
```

### PR DESCRIPTION
**Pull request by Moosems**
Fixes #102

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - run: brew install python@3.9    # https://github.com/actions/setup-python/issues/58
+    - run: brew install python-tk
```
Comment (Akuli) [inline .github/workflows/build.yml]:
```suggestion
    - run: brew install python-tk@3.9
```

**Comment by Akuli** [comment]
Turns out that the builds are broken in general. I'll fix them in #104

**Comment by Akuli** [comment]
@Moosems Please check whether these work:
- Release v0.0.11 that was created a few minutes ago. You can find it on the right sidebar at https://github.com/ThePhilgrim/minesweeper. It includes my pull request but nothing from this PR.
- This PR's build. To access it, click "Details" next to mac_build below, then "Summary", then the mac-dmg artifact near the bottom of the page.

**Comment by Moosems** [comment]
> Turns out that the builds are broken in general

How so?

**Comment by Moosems** [comment]
Both give the following issue:
```yaml
/Volumes/Minesweeper/minesweeper.app/Contents/MacOS/minesweeper ; exit;
Moosems-MacBook-Pro:~ Moosems$ /Volumes/Minesweeper/minesweeper.app/Contents/MacOS/minesweeper ; exit;
[3222] Error loading Python lib '/Volumes/Minesweeper/minesweeper.app/Contents/MacOS/Python': dlopen: dlopen(/Volumes/Minesweeper/minesweeper.app/Contents/MacOS/Python, 10): Symbol not found: _preadv
  Referenced from: /Volumes/Minesweeper/minesweeper.app/Contents/MacOS/Python (which was built for Mac OS X 11.0)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Volumes/Minesweeper/minesweeper.app/Contents/MacOS/Python
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

**Comment by Moosems** [comment]
I made an issue on pyinstaller to get some help with backwards compatibility and they came up with [this](https://github.com/sabnzbd/sabnzbd/blob/cbeab4dd55a12499ff3cc32d1fca440e0f24cfa7/.github/workflows/build_release.yml#L65). We could try to replicate what they do here

**Comment by Moosems** [comment]
See PyInstallerBuilds.

**Pull request closed without merging**